### PR TITLE
Static binary

### PR DESCRIPTION
**Pull request by christopherL91**
This is now a static binary

### CODE REVIEW

**Comment by christopherL91** [comment]
Have tried to make it "fully" static but there are some C dependencies that needs to be fixed in order to make it happen. If everything was just Go then it would have been very easy to produce binaries for every available platform. Maybe you have more experience using cgo in cross compiling?

**Comment by cortex** [comment]
Thank you, this looks good! I think building with this set of flags should do it: `go build -ldflags "-linkmode external -extldflags -static"` I only got half-way with this on my machine because I have to rebuild lots of packages for static builds, but perhaps you will have better luck.

File: makefile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+	go build
+
+.PHONY: install
+install:
+	go install
+
+.PHONY: clean
+clean:
+	rm qrc.go
+	go clean
```
Comment (cortex) [inline makefile:14]: "Makefile" is the name recommended by GNU Make, let's stick to that. I also think we don't need .PHONY targets unless we could have name collisions. Or is this good practice?

File: main.go [hunk 2]
```diff
@@ -14,6 +12,10 @@ import (
 	"path/filepath"
 	"strings"
 	"time"
+
+	"github.com/atotto/clipboard"
+	"github.com/proglottis/gpgme"
+	"gopkg.in/qml.v1"
```
Comment (cortex) [inline main.go:18]: This is goimports ordering I suppose, I should fix my editor config =)

File: makefile (hunk 1 above)
Comment (christopherL91) [inline makefile:14]: No you're absolutely right, my bad

**Pull request merged**